pkg/daemon/daemon.go

```diff
@@ -280,6 +280,7 @@ func NewUnikDaemon(config config.DaemonConfig) (*UnikDaemon, error) {
 	}
 
 	_compilers[compilers.OSV_JAVA_XEN] = osvJavaXenCompiler
+	_compilers[compilers.OSV_JAVA_OPENSTACK] = osvJavaXenCompiler
```

Comment:
And one last time.

Additionally, please make a comment about the `Xen` stuff.